README.md

```diff
@@ -17,6 +17,15 @@ A web application for providing access to Digital Objects in SDR Storage.
   ./bin/test.sh
   ```
 
+### Note for osx users
+the current bundle requires eventmachine 1.0.7. This won't build on a modern mac
+without specifying where the ssl libraries are:
+
+```
+gem install eventmachine -v '1.0.7' -- --with-cppflags=-I/usr/local/opt/openssl/include
+```
+
```

Comment:
Take all of this out; I think this belongs in the eventmachine docs, not here.  If anyone has problems installing it, they should go to the eventmachine docs to find this.  If necessary, put this in the Gemfile instead of the README.